pandas/_libs/tslibs/strptime.pyx

```diff
@@ -0,0 +1,631 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+"""Strptime-related classes and functions.
+
+CLASSES:
+    LocaleTime -- Discovers and stores locale-specific time information
+    TimeRE -- Creates regexes for pattern matching a string of text containing
+                time information
+
+FUNCTIONS:
+    _getlang -- Figure out what language is being used for the locale
+    strptime -- Calculates the time struct represented by the passed-in string
```

Comment:
I know you copy pasted this, but this does not really suite for the module docstring, as this is only for part of the file. 

So I would either move this inline to where those functions/classes are defined (as it was in tslib), or update it to reflect the actual file contents

If I understand correctly `array_strptime` is the only 'public' (for pandas, as used outside this file) function? If so, I would focus on that in the module docstring

Reply:
There are a handful of style issues I'd like to circle back to (e.g. mixed_camel_Case), will add this to the list.  For the time being I'm _really_ eager to wrapping this up.

Reply:
This is not really a style issue, but how you copy pasted things. If you want to clean-up later, I would just put this in the middle close of the file close to the relevant code for now (like it was in tslib)

Reply:
Not sure I understand.  Are you OK with this being addressed in a follow-up PR?

Reply:
I would rather you fix the docs now. just move these docs down to where they are used, and put array_strptime up front

Reply:
@jbrockmendel I see you integrated them into the class/functions docstrings, which is good, but I would still include a comment where LocaleTime starts (the place where previously the comment was) to make it clear that this part is some kind of vendored code (it just comes from the standard library: https://github.com/python/cpython/blob/master/Lib/_strptime.py)